src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java

```diff
@@ -28,11 +29,7 @@
 import java.io.FileInputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
I guess for a minimal diff this change can be reverted then ;)

Reply:
whoops